modules/client/src/test/java/org/apache/ignite/client/ClientComputeTest.java

```diff
@@ -197,10 +197,10 @@ public void testExecuteColocatedThrowsTableNotFoundExceptionWhenTableDoesNotExis
 
             var ex = assertThrows(CompletionException.class,
                     () -> client.compute().executeAsync(
-                            JobTarget.colocated("\"bad-tbl\"", key), JobDescriptor.builder("job").build(), null).join());
+                            JobTarget.colocated("\"bad_tbl\"", key), JobDescriptor.builder("job").build(), null).join());
```

Comment:
Please revert, or add a test that quoted names work on client and embedded.

Reply:
done, thanks